### ISSUE TITLE
DMR and Nexedge Trunk Tracking

### ISSUE THREAD

**Issue by JoeyAbsi** [moneriomaa/openear#58]
OpenEar project is a very good software! It can decode DMR, Nexedge and other type of commincations.

It would be a even better if it could trunk track and decode DMR and Nexedge systems.

**Comment by JoeyAbsi** [comment]
Also, it would be good if OpenEar could display Radio ID, Talkgroup ID, Color Codes and Time Slots.

**Comment by argamesinpaul** [comment]
Yes and have the possibility of 1 that can follow the conversations in cap + with a single rtl
